moped-editor/src/styles/reusableStyles.js

```diff
@@ -11,7 +11,7 @@ export const fieldLabel = (theme) => ({
 
 export const fieldLabelText = (theme) => ({
   width: "calc(100% - 2rem)",
-  paddingLeft: theme.spacing(0.5),
+  padding: theme.spacing(0.5),
```

Comment:
This adds padding on all sides of the field values in the summary card for a bit more spacing, and this also helps align the value of the eCAPRIS id and the "Copy link" button which position awkwardly without this change.